contrib/templates/data-engineering/template/{{.project_name}}/conftest.py

```diff
@@ -0,0 +1,37 @@
+# conftest.py is used to configure pytest.
+# This file is in the root since it affects all tests through this bundle.
```

Comment:
This should be expanded on these tests existing under `assets` and being dynamically added.